rajawali/src/main/java/org/rajawali3d/loader/LoaderOBJ.java

```diff
@@ -94,8 +94,12 @@
     protected final String DIFFUSE_COLOR = "Kd";
     protected final String DIFFUSE_TEX_MAP = "map_Kd";
 
+    private boolean mNeedToRenameMtl = true;
```

Comment:
Don't set it to `true` here and then `false` in the constructor. Pick one place to initialize it.